ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -176,6 +176,19 @@ export type wireframeButton = wireframeInputBase & {
     value?: string
 }
 
+export type wireframeProgress = wireframeInputBase & {
+    inputType: 'progress'
+    progressType: 'horizontal' | 'circular'
```

Comment:
should this be part of `style` instead? also fine to keep it here, but feels like part of it.